test/extensions/filters/http/header_to_metadata/header_to_metadata_filter_test.cc

```diff
@@ -42,47 +72,24 @@ class HeaderToMetadataTest : public testing::Test {
 
   void initializeFilter(const std::string& yaml) {
     envoy::extensions::filters::http::header_to_metadata::v3::Config config;
-    TestUtility::loadFromYaml(yaml, config);
+    if (!yaml.empty()) {
```

Comment:
is this change necessary? mostly curious, i dont mind it

Reply:
Fixed -- it wasn't necessary (the problem was I was using "" as an empty YAML config, instead of "{}").